security_monkey/auditors/iam/iam_policy.py

```diff
@@ -210,10 +210,10 @@ def library_check_attached_managed_policies(self, iam_item, iam_type):
                 mp_arn = mp_item.config.get('arn', mp_item.config.get('Arn'))
                 if mp_arn == item_mp_arn:
                     found = True
-                    for issue in mp_item.db_item.issues:
```

Comment:
Disabling managed policy checks?

Personally, I don't have an issue with it since the issue system is mostly broken. But just wanted to point out.

Reply:
Correct, I'm glad you are pointing this out, this was done intentionally as part of the python3 parity to move forward, but I see value in the function and will update this with the next platform update returning this to functionality with the boto updates.